understanding/20/audio-description-prerecorded.html

```diff
@@ -7,39 +7,37 @@
 </head>
 <body>
    <h1>Understanding Audio Description (Prerecorded)</h1>
-   
+
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
          <dt>Goal</dt><dd>Videos can be played with audio descriptions.</dd>
          <dt>What to do</dt><dd>Provide a synchronized spoken description of the visual content in videos.</dd>
          <dt>Why it's important</dt><dd>People who cannot see or understand the visual content can hear about it while playing videos.</dd> 
       </dl>
-
    </section>
-   
+
    <section id="intent">
       <h2>Intent of Audio Description (Prerecorded)</h2>
-      
-      
       <p>The intent of this Success Criterion is to provide people who are blind or visually
          impaired access to the visual information in a synchronized media presentation. The
          audio description augments the audio portion of the presentation with the information
-         needed when the video portion is not available. During existing pauses in dialogue,
-         audio description provides information about actions, characters, scene changes, and
-         on-screen text that are important and are not described or spoken in the main sound
-         track.
+         needed when the video portion is not available. Audio description provides information about actions,
+         characters, scene changes, and on-screen text that are important and are not described
+         or spoken in the main sound track. Audio description is added during existing
+         pauses in the dialogue.
       </p>
-      
       <div class="note">
-         
+         <p>This Success Criterion applies to all video content that contains <q>important visual details that cannot be understood from the main soundtrack alone</q>. Content is not exempted if the original audio offers no pauses in the dialog where audio descriptions can be provided – it fails this Success Criterion. Similarly, where all pauses in the dialogue contain audio descriptions, yet additional important visual information has not been described, it fails this Success Criterion.</p>
```

Comment:
While I'm fine saying that a video with no audio descriptions cannot pass 1.2.3, as with Bruce's previous comment, I disagree with the latter part of this new paragraph:
"Similarly, where all pauses in the dialogue contain audio descriptions, yet additional important visual information has not been described, it fails this Success Criterion."